common/src/main/java/com/sequenceiq/cloudbreak/ccm/endpoint/ServiceEndpointRequest.java

```diff
@@ -58,6 +59,11 @@ public String getTargetInstanceId() {
                 return serviceFamily;
             }
 
+            @Override
+            public Optional<Integer> getPort() {
+                return Optional.ofNullable(port);
```

Comment:
Is it possible to make a member variable when instantiating a class like this? I figure you could do

```
private Optional<Integer> optionalPort;

...

return optionalPort;
```

Instead. If not, no worries, just a thought.

Reply:
I don't think so.